alt1/scripts/notifications.ts

```diff
@@ -20,3 +29,109 @@ export function showToast(message: string, type: "error" | "success" = "success"
         setTimeout(() => toast.remove(), 500); // Remove from DOM
     }, 3000);
 }
+
+export function notifyEvent(event: EventRecord): void {
+    // early return if we aren't in alt1
+    if (!window.alt1) {
+        return;
+    }
+
+    const suppressToday = localStorage.getItem("toggleNotificationsToday") === "true";
+    if (suppressToday) {
+        return;
+    }
```

Comment:
If suppressed and not receiving events, maybe remove the toolbar text as well? More info in the `showTitleBarText` function comment below.